docs/terms_of_service.md

```diff
@@ -0,0 +1,84 @@
+Terms of Service
+================
+
+Terms of Service allows Box Admins to configure a custom Terms of Service for end users to
+accept/re-accept/decline for custom applications
+
+
+Create a Terms of Service
+-------------------------
+
+A terms of service can be created in an enterprise. Please note that only two can be created. One external
+and one managed. If a terms of service already exists please use the update call to change the current
+terms of service.
+
+You can create a custom terms of service by using `client.create_terms_of_service(status, tos_type, text)`.
+
+```python
+from boxsdk.object.terms_of_service import TermsOfServiceType, TermsOfServiceStatus
+terms_of_service = client.create_terms_of_service(TermsOfServiceStatus.ENABLED, TermsOfServiceType.MANAGED, 'Example Text')
+```
+
+Edit a Terms of Service
+-----------------------
+
+To update a terms of service, use `terms_of_service.update_info()`.
+
+```python
+tos_id = '1234'
+update_object = {'text': 'New Text'}
+updated_tos = client.terms_of_service(tos_id).update_info(update_object)
+```
+
+Get Terms of Service
+--------------------
+
+To retrieve information about a terms of service, use `terms_of_service.get()`.
+
+```python
+tos_id = '1234'
+terms_of_service = client.terms_of_service(tos_id).get()
+```
+
+List Terms of Service
+---------------------
+
+You can retrieve an iterable of terms of service in your enterprise by calling `client.get_terms_of_services(tos_type=None, limit=None)`.
+
+```python
+terms_of_services = client.get_terms_of_services()
+for terms_of_service in terms_of_services:
+    # Do something
+```
+
+Accept or Decline a Terms of Service for New User
+-------------------------------------------------
+
+For new users you can accept or decline a terms of service by calling `terms_of_service.accept(user=None)` or `terms_of_service.reject(user=None)`.
+
+```python
+user = client.user('1234')
+user_status = client.terms_of_service('11111').accept(user)
+```
+
+
+Accept or Decline a Terms of Service for Existing User
+------------------------------------------------------
+
+For an existing user you can accept or decline a terms of service by calling `terms_of_service.set_user_status(is_accepted, user=None)` to accept or to reject.
+
+```python
+user = client.user('22222')
+user_status = client.terms_of_service('11111').set_user_status(is_accepted=True, user=user)
```

Comment:
This isn't really the best example for this; if you know the user has already accepted or declined the ToS you should not use the convenience method this way since it will make an extra API call